README.md

```diff
@@ -333,6 +333,10 @@ Basic flags:
 Platform flags:
 - :whale: `--platform=(amd64|arm64|...)`: Set platform
 
+Isolation flags:
+- :whale: :blue_square: :nerd_face: `--isolation=(default|host)`: Used on Windows to change process isolation level. Valid values are `default` and `host`. `default` runs process isolated containers.  `host` runs Host Process containers.  Host process containers inherit permissions from containerd process unless `--user` is specified then will start with user specified.  The user specified must be present on the host.  `host` requires Containerd 1.6+. Not implemented for Linux.
+
```

Comment:
Wasn't the Docker option for process isolated "process"? I think for Docker Desktop it defaulted to hyper-v if you didn't supply anything, and on server it defaulted to process. Regardless of that we should probably stick to the same naming as docker (unless I missed the discussion for naming this default 😆)

Reply:
yes, I should probably have `process` as an explicit support flag but since hyper-v isn't really support fully in containerd we can default that.

The `default` flag uses whatever was set on the docker daemon via `--exec-opt isolation=hyperv` option: https://docs.docker.com/engine/reference/commandline/run/#specify-isolation-technology-for-container---isolation

I don't think containerd has this setting but the equivalent would be `default_runtime_name`?